src/java.base/windows/native/libjava/ProcessImpl_md.c

```diff
@@ -379,13 +380,21 @@ Java_java_lang_ProcessImpl_create(JNIEnv *env, jclass ignored,
                 if (!(*env)->ExceptionCheck(env)) {
                     jlong *handles = (*env)->GetLongArrayElements(env, stdHandles, NULL);
                     if (handles != NULL) {
-                        ret = processCreate(
-                            env,
-                            pcmd,
-                            penvBlock,
-                            pdir,
-                            handles,
-                            redirectErrorStream);
+                        // Copy command line to mutable char buffer; CreateProcessW may modify it
+                        jsize cmdLen = (*env)->GetStringLength(env, cmd);
+                        WCHAR *pcmdCopy = (WCHAR*)malloc(cmdLen * sizeof(WCHAR));
```

Comment:
Should this include null terminator, as it is interpreted as `LPWSTR` which is null-terminated?

Reply:
It seems like it should now.  Java strings are not null terminated and any null termination would have accidental.
But it has worked for a long time (2013) without explicit Unicode null termination.